src/library/scala/collection/convert/ImplicitConversions.scala

```diff
@@ -17,76 +17,122 @@ import JavaConverters._
+   */
   implicit def `enumeration asJava`[A](it: Iterator[A]): ju.Enumeration[A] = asJavaEnumeration(it)
 
-  /** Implicitly converts a Scala `Iterable` to a Java `Iterable`. See [[asJavaIterable]]. */
+  /** Implicitly converts a Scala `Iterable` to a Java `Iterable`.
+   *  @see [[AsJavaConverters.asJavaIterable]]
+   */
   implicit def `iterable asJava`[A](i: Iterable[A]): jl.Iterable[A] = asJavaIterable(i)
 
-  /** Implicitly converts a Scala `Iterable` to an immutable Java `Collection`. See [[asJavaCollection]]. */
+  /** Implicitly converts a Scala `Iterable` to an immutable Java `Collection`.
+   *  @see [[AsJavaConverters.asJavaCollection]]
+   */
   implicit def `collection asJava`[A](it: Iterable[A]): ju.Collection[A] = asJavaCollection(it)
 
-  /** Implicitly converts a Scala mutable `Buffer` to a Java `List`. See [[bufferAsJavaList]]. */
+  /** Implicitly converts a Scala mutable `Buffer` to a Java `List`.
+   *  @see [[AsJavaConverters.bufferAsJavaList]]
+   */
   implicit def `buffer AsJavaList`[A](b: mutable.Buffer[A]): ju.List[A] = bufferAsJavaList(b)
 
-  /** Implicitly converts a Scala mutable `Seq` to a Java `List`. See [[mutableSeqAsJavaList]]. */
+  /** Implicitly converts a Scala mutable `Seq` to a Java `List`.
+   *  @see [[AsJavaConverters.mutableSeqAsJavaList]]
+   */
   implicit def `mutableSeq AsJavaList`[A](seq: mutable.Seq[A]): ju.List[A] = mutableSeqAsJavaList(seq)
 
-  /** Implicitly converts a Scala `Seq` to a Java `List`. See [[seqAsJavaList]]. */
+  /** Implicitly converts a Scala `Seq` to a Java `List`.
+   *  @see [[AsJavaConverters.seqAsJavaList]]
+   */
   implicit def `seq AsJavaList`[A](seq: Seq[A]): ju.List[A] = seqAsJavaList(seq)
 
-  /** Implicitly converts a Scala mutable `Set` to a Java `Set`. See [[mutableSetAsJavaSet]]. */
+  /** Implicitly converts a Scala mutable `Set` to a Java `Set`.
+   *  @see [[AsJavaConverters.mutableSetAsJavaSet]]
+   */
   implicit def `mutableSet AsJavaSet`[A](s: mutable.Set[A]): ju.Set[A] = mutableSetAsJavaSet(s)
 
-  /** Implicitly converts a Scala `Set` to a Java `Set`. See [[setAsJavaSet]]. */
+  /** Implicitly converts a Scala `Set` to a Java `Set`.
+   *  @see [[AsJavaConverters.setAsJavaSet]]
+   */
   implicit def `set AsJavaSet`[A](s: Set[A]): ju.Set[A] = setAsJavaSet(s)
 
-  /** Implicitly converts a Scala mutable `Map` to a Java `Map`. See [[mutableMapAsJavaMap]]. */
+  /** Implicitly converts a Scala mutable `Map` to a Java `Map`.
+   *  @see [[AsJavaConverters.mutableMapAsJavaMap]]
+   */
   implicit def `mutableMap AsJavaMap`[A, B](m: mutable.Map[A, B]): ju.Map[A, B] = mutableMapAsJavaMap(m)
 
-  /** Implicitly converts a Scala mutable `Map` to a Java `Dictionary`. See [[asJavaDictionary]]. */
+  /** Implicitly converts a Scala mutable `Map` to a Java `Dictionary`.
+   *  @see [[AsJavaConverters.asJavaDictionary]]
+   */
   implicit def `dictionary asJava`[A, B](m: mutable.Map[A, B]): ju.Dictionary[A, B] = asJavaDictionary(m)
```

Comment:
Out of scope for this PR but the naming convention is not followed on this method. To match it should have been

    mutableMap AsJavaDictionary